build_projects/dotnet-cli-build/TestTargets.cs

```diff
@@ -83,7 +83,7 @@ public static BuildTargetResult RestoreTestAssetPackages(BuildTargetContext c)
             CleanNuGetTempCache();
 
             var dotnet = DotNetCli.Stage2;
-            dotnet.Restore("--verbosity", "verbose")
+            dotnet.Restore("--verbosity", "verbose", "--disable-parallel")
```

Comment:
I was hitting some global.json access problems in docker when we didn't have the disable-parallel here.